File: l1/Lab1 forms/WinFormsApp1/Result.cs
namespace Lab1
{
    internal class Result
    {
        public int total_weight;
        public int total_value;
        public List<Item> items;
        public Result()
        {
            this.total_weight = 0;
            this.total_value = 0;
            this.items = new List<Item>();
        }

        public override string ToString()
        {
            string res = "Total weight: " + this.total_weight + " Total_value: " + this.total_value + "\nItems:\n";
            foreach (Item item in items)
            {
                res += item.ToString() + "\n";
            }

            return res;
        }
    }
}

File: l5/src/main/java/org/example/Result.java
package org.example;

import java.util.ArrayList;
import java.util.List;

public class Result {
    int total_weight = 0;
    int total_value = 0;
    List<item> items = new ArrayList<item>();

    public void add_item(item item) {
        items.add(item);
        total_weight += item.weight;
        total_value += item.value;
    }

    @Override
    public String toString() {
        String str = "";

        str += "Total weight: " + total_weight + "\n" + "Total value: " + total_value + "\n";

        for (item i : items) {
            str += i.toString() + "\n";
        }

        return str;
    }
}
